modules/client-handler/src/integrationTest/java/org/apache/ignite/client/handler/ItClientHandlerTest.java

```diff
@@ -59,32 +60,36 @@ public class ItClientHandlerTest extends BaseIgniteAbstractTest {
     @InjectConfiguration(rootName = "security")
     private SecurityConfiguration securityConfiguration;
 
-    @BeforeEach
-    public void setUp(TestInfo testInfo) {
-        testServer = new TestServer(null, securityConfiguration);
-        serverModule = testServer.start(testInfo);
-        serverPort = serverModule.localAddress().getPort();
-    }
+    @InjectConfiguration
+    private ClientConnectorConfiguration clientConnectorConfiguration;
 
-    @AfterEach
-    public void tearDown() throws Exception {
-        serverModule.stop();
-        testServer.tearDown();
-    }
+    @InjectConfiguration
+    private NetworkConfiguration networkConfiguration;
 
     @Test
-    void testHandshakeInvalidMagicHeaderDropsConnection() throws Exception {
+    void testHandshakeInvalidMagicHeaderDropsConnection(TestInfo testInfo) throws Exception {
+        testServer = new TestServer(null, securityConfiguration, clientConnectorConfiguration, networkConfiguration);
```

Comment:
Why did you do this? All tests have a lot of copy-paste now. If this is because you wanted to provide a different clientConnectorConfiguration, then I would suggest to move those tests into a different class